packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -212,6 +220,12 @@ class MessageList extends Widget<Properties> {
       year: 'numeric',
     }).replace(/[/-]/g, '.');
 
+    const { dayHeaderFormat } = this.option();
+
+    if (dayHeaderFormat) {
+      headerDate = dateLocalization.format(deserializedDate, dayHeaderFormat);
```

Comment:
Why we not use one of predefined format 'shortdate' for examle

dateLocalization.format(date, messageTimestampFormat ?? 'shortdate');
```suggestion
      headerDate = dateLocalization.format(deserializedDate, dayHeaderFormat);
```
https://github.com/Zedwag/DevExtreme/blob/9614145c4838d3944504d7fb2af44433f4d1d4e4/packages/devextreme/js/localization/date.js#L19

Instead of this we use `date.toLocaleDateString()`

Well, 
1) it supports all browsers that we support
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString
2) toLocaleDateString returns a string with a language-sensitive representation it means that it shouldn't contain '-' that you replace on the line 221.

<img width="672" alt="image" src="https://github.com/user-attachments/assets/810c7bf8-01ef-4fc1-a25d-b7591de815a6">
<img width="348" alt="image" src="https://github.com/user-attachments/assets/83b0a4f1-e3db-4296-a0db-c29fc89dc011">

Please look into this matter in more detail to ensure that we are not making a mistake

Reply:
Yeah, i believe we should always use `dateLocalization.format` with some format.
Maybe we even should get rid of `null` in type and set some default format instead of this - I think it'll be much clearer

Reply:
ed2574f9d5f3c5a301c284eb8acaba16c73321ad